src/js/program/program-controller.js

```diff
@@ -313,23 +346,32 @@ class ProgramController extends Events {
     /**
      * Loads the next playlist item in the background.
      * @param {Item} item - The playlist item to load.
+     * @param {number} index - The playlist item promise index.
      *
      * @returns {void}
      */
-    backgroundLoad(item) {
-        const { background } = this;
-        const source = getSource(item);
+    backgroundLoad(item, index) {
+        const { api, background } = this;
 
-        background.setNext(item, this._setupMediaController(source)
+        const loadPromise = this.getItemPromise(index, api).preload()
+            .then((playlistItem) => {
+                if (playlistItem && playlistItem !== item) {
+                    this.setPlaylistItem(index, playlistItem);
+                    background.updateNext(playlistItem);
+                }
+                const source = getSource(playlistItem);
+                return this._setupMediaController(source);
+            })
             .then(nextMediaController => {
                 nextMediaController.activeItem = item;
                 nextMediaController.preload();
                 return nextMediaController;
             })
             .catch(() => {
                 background.clearNext();
-            })
-        );
+            });
```

Comment:
mismatched tabs?

Reply:
No. I moved the `loadPromise` promise chain out of `background.setNext(item, loadPromise);`